src/genie/libs/parser/nxos/show_lldp.py

```diff
@@ -192,7 +192,7 @@ class ShowLldpNeighborsDetailSchema(MetaParser):
                                     }
                                 },
                                 'management_address_v4': str,
-                                'management_address_v6': str,
+                                Optional('management_address_v6'): str,
```

Comment:
Is this new key being set by any regex in the parser? 
I don't see it in the parsed output example provided

Reply:
It is not a new key, it already exists 
`p11 = re.compile(r'^Management +Address +IPV6: +(?P<mgmt_address_ipv6>.+)$')`

and it will be set here:

```python
# Management Address IPV6: not advertised
m = p11.match(line)
if m:
  group = m.groupdict()
  sub_dict.update({'management_address_v6': group['mgmt_address_ipv6']})

  continue
```

The `management_address_v6` field was mandatory before my change, but we got results from devices where no `management_address_v6` was set (as I mentioned in my opening post). Because of that I set the field to be optional.

Reply:
Ah right, apologies I missed that